apps/server/src/modules/lesson/index.ts

```diff
@@ -1,5 +1,5 @@
-export * from './lesson.module';
-export * from './types/lesson-copy-parent.params';
-export * from './types/lesson-copy.params';
-export { LessonService, LessonCopyService, EtherpadService } from './service';
+export { EtherpadService, LessonCopyService, LessonService } from './domain/service';
+export * from './domain/types/lesson-copy-parent.params';
+export * from './domain/types/lesson-copy.params';
```

Comment:
```suggestion
export { LessonCopyParams, LessonCopyParentParams } from './domain';
```
Sollen hier die direkten Exporte verwendet werden und nicht über die Typen und Barrel-Files selber?